src/effects/VST/VSTEffect.cpp

```diff
@@ -3551,6 +3551,15 @@ bool VSTEffectWrapper::StoreSettings(const VSTEffectSettings& vst3settings) cons
    return false;
 }
 
+bool VSTEffect::TransferDataToWindow(const EffectSettings& settings)
+{
+   if (!StoreSettings(GetSettings(settings)))
+      return false;
+
+   // TODO (?): update of the GUI controls 
```

Comment:
The commit that introduces this function should also call `RefreshParameters()` which is a step done in the existing functions that load user and factory presets and that import.

Maybe you did this in a later commit I haven't reviewed yet?  But you can do it in this early commit to make the function complete.

Reply:
Now I have looked at the whole PR and I see this wasn't done.  The task breakdown said

"Define TransferDataToWindow (to become UpdateUI of a validator later), calling StoreSettings; remove parts of LoadSettings that duplicate this"

So, to make that step complete, add the `RefreshParameters()` call and also remove the call in three places, all in the same commit.

Reply:
You are right, I forgot this. Now done.